pkg/job/delete_all.go

```diff
@@ -0,0 +1,49 @@
+package job
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "mayadata.io/d-operators/types/job"
+	"openebs.io/metac/dynamic/clientset"
+)
+
+func (r *TaskRunner) deleteAll() (*types.TaskStatus, error) {
```

Comment:
Please add Unit Tests for all the changes or additions.

Reply:
I will add unit test for these all changes In my coming PR @AmitKumarDas